src/lab6.py

```diff
@@ -309,6 +276,51 @@ def __repr__(self):
         return "InlineLayout(x={}, y={}, width={}, height={})".format(
             self.x, self.y, self.width, self.height)
 
+class BlockLayout:
```

Comment:
Did this class change in lab6 but we failed to notice it?

Reply:
Ah, no. This is subtle. Since `BlockLayout` creates `InlineLayout` objects, if you import `BlockLayout` from lab 5 it will use lab 5's `InlineLayout` even if in lab 6 we define a new one. (This is an argument, perhaps, for construing the layout tree separately from layout.)

Reply:
Ok, you're fixing that bug then. Got it.